lib/rails-jquery-autocomplete/orm/active_record.rb

```diff
@@ -24,27 +24,42 @@ def active_record_get_autocomplete_items(parameters)
 
         items = (::Rails::VERSION::MAJOR * 10 + ::Rails::VERSION::MINOR) >= 40 ? model.where(nil) : model.scoped
 
-        scopes.each { |scope| items = items.send(scope) } unless scopes.empty?
-
         items = items.select(get_autocomplete_select_clause(model, method, options)) unless options[:full_model]
         items = items.where(get_autocomplete_where_clause(model, term, method, options)).
             limit(limit).order(order)
         items = items.where(where) unless where.blank?
 
+        if options[:unique]
+          scopes << -> {
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Use the `lambda` method for multi-line lambdas.